### PR TITLE
Update admin_sanitization.md

### PR DESCRIPTION
**Pull request by brittainmark**
Corrected MD for FILE_DIR_REGEX added FLOAT_VALUE_REGEX

### CODE REVIEW

**Comment by brittainmark** [comment]
Also noted missing are 
SANITIZE_EMAIL_AUDIENCE
PRODUCT_URL_REGEX 
FILE_PATH_OR_URL_REGEX Not used
should they also be added?

**Comment by scottcwilson** [comment]
I'm ok with adding anything that's used.  But the point of this document is not necessarily to be exhaustive but rather to get people started in understanding the sanitization layer.

File: content/dev/code/admin_sanitization.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ Zen Cart defines the following default case-insensitive sanitizers:
 
     POST values only 
     
-    uses `[^0-9a-z.!@#$%^& ()`_+-~/\\]` regex
+    uses ``[^0-9a-z.!@#$%^& ()`_+-~/\\]`` regex
```
Comment (drbyte) [inline content/dev/code/admin_sanitization.md:146]: Why the change to double back-ticks?

Comment (scottcwilson) [reply content/dev/code/admin_sanitization.md:146]: I was surprised by this too but when I looked at it in the preview, it looked the same.  @brittainmark is this needed or should it be removed here and in your other PR?
